docs/user_guide/models/destvi.md

```diff
@@ -107,11 +107,11 @@ $p_g$ is the rate parameter for the negative binomial distribution.
 To avoid the latent variable $\gamma_s^c$ from incorporating variation attributed to experimental
 assay differences, we assign an empirical prior informed by the scLVM and a corresponding set of
 cells of the same cell type in the scRNA-seq dataset. To compute this function, we subcluster the latent space of the
-scLVM for each cell type to k cell type specific clusters. For each cluster we compute an empirical mean and variance.
+scLVM for each cell type to K cell type specific clusters. For each cluster we compute an empirical mean and variance.
 The loss is weighted by the probability of a random cell from this cell type to be in the respective cluster in the
 scRNA-seq dataset (mixture probability, $m_{kc}$).
-Above, $\{u_{kc}\}_{k=1}^K$ designates a set of cells from cell type $c$ in the scRNA-seq dataset, and
-$q_\Phi$ designates the variational distrbution from the scLVM.
+Above, $\{u_{kc}\}$ designates the set of cell type specific subclusters from cell type $c$ in the scRNA-seq dataset, and
+$q_\Phi$ designates the empirical normal distribution from cluster mean and variance.
```

Comment:
I'm not sure this change adds any clarity, I think it needs to be more specific

Reply:
I think it's as specific as beforehand. However, we don't use a set of cells anymore and that is stated here several times. Do you have any suggestions what else to write?